app/src/main/kotlin/com/wire/android/ui/home/conversations/DownloadedAssetDialog.kt

```diff
@@ -28,19 +25,6 @@ fun DownloadedAssetDialog(
         val assetSize = downloadedAssetDialogState.assetSize
         val messageId = downloadedAssetDialogState.messageId
 
-        // Flow to get write to external storage permission
-        val requestWriteToExternalStorageRequestFlow = WriteToExternalStorageRequestFlow(
```

Comment:
will removing this brake is for Android < 13 ?

Reply:
Nope. It turned out we don't need to request write permissions when writing AR internal files to download folder when using the content provider also for Android < 13. Actually this is how we were already doing it in the GalleryViewModel.
I tested it on Android 12 and 13 and everything seems fine.